src/core/util/byte_stream.cc

```diff
@@ -100,6 +101,11 @@ void OutputByteStream::ProduceData(std::size_t amount) {
 }
 
 void OutputByteStream::WriteData(const std::uint8_t* data, std::size_t len) {
+  if (!data || !len)
+    {
+      LOG(debug) << "OutputByteStream: skip null data or size";
+      return;
+    }
```

Comment:
Yeah, that was retarded, sorry. After i agreed with your idea, i just copy/pasted your previous instruction. And after that, i changed the unit-test accordingly with the code (u can never be wrong this way :man_facepalming: )